uaclient/cli/cve.py

```diff
@@ -0,0 +1,174 @@
+                                name=binary_pkg,
+                                fix_status=cve.fix_status,
+                                fix_origin=cve.fix_origin,
+                                fix_version=cve.fix_version,
+                            )
+                        )
+                        break
+
+        return rows
+
+
+def _format_affected_pkgs(affected_pkgs):
+    formatted_rows = []
+
+    for affected_pkg in affected_pkgs:
+        if affected_pkg.fix_status == "fixed":
+            formatted_rows.append(
+                [
+                    "{}:".format(affected_pkg.name),
+                    affected_pkg.fix_status,
+                    "({})".format(affected_pkg.fix_origin),
+                    affected_pkg.fix_version,
+                ]
+            )
+        else:
+            formatted_rows.append(
+                [
+                    "{}:".format(affected_pkg.name),
+                    affected_pkg.fix_status,
+                    "",
+                    "",
+                ]
+            )
+
+    return formatted_rows
+
+
+def action_cve(args, *, cfg, **kwargs):
+    cve_vulnerabilities = _get_cve_vulnerabilities(args, cfg=cfg)
+
+    if args.cve not in cve_vulnerabilities.cves:
+        print(messages.CLI_CVE_NOT_AFFECTED.format(issue=args.cve))
+        return
+
+    cve_info = cve_vulnerabilities.cves[args.cve]
+    affected_pkgs_rows = _format_affected_pkgs(
+        _get_affected_pkgs(cve_vulnerabilities, cve_info, args.cve)
+    )
+    affected_pkgs_table = Table(rows=affected_pkgs_rows)
+
+    print("name:            {}".format(args.cve))
+    print("public-url:      https://ubuntu.com/security/{}".format(args.cve))
+    print(
+        "published-at:    {}".format(
+            cve_info.published_at.strftime("%Y-%m-%d")
+        )
+    )
+    print(
+        "cve-cache-date:  {}".format(
+            cve_vulnerabilities.vulnerability_data_published_at.strftime(
+                "%Y-%m-%d"
+            )
+        )
+    )
+    print(
+        "apt-cache-date:  {}".format(
+            cve_vulnerabilities.apt_updated_at.strftime("%Y-%m-%d")
+        )
+    )
+    print(
+        "priority:        {}".format(
+            cli_util.colorize_priority(cve_info.priority)
+        )
+    )
+    print("cvss-score:      {}".format(cve_info.cvss_score))
+    print("cvss-severity:   {}".format(cve_info.cvss_severity))
+    print("description: |")
+    print(
+        "{}".format(
+            "\n".join(
+                textwrap.wrap(
+                    cve_info.description,
+                    width=80,
+                    break_long_words=False,
+                    break_on_hyphens=False,
+                    initial_indent="  ",
+                    subsequent_indent="  ",
+                )
+            )
+        )
+    )
+
+    if cve_info.notes:
+        print("notes:")
+        for note in cve_info.notes:
+            print("  - {}".format(note.replace("\n", "\n    ")))
```

Comment:
Last nit suggestion on this section I promise :)

```suggestion
    print("description: |")
    print(
        textwrap.fill(
            cve_info.description,
            width=defaults.PRINT_WRAP_WIDTH,
            break_long_words=False,
            break_on_hyphens=False,
            initial_indent="  ",
            subsequent_indent="  ",
        )
    )

    if cve_info.notes:
        print("notes:")
        for note in cve_info.notes:
            print(
                textwrap.fill(
                    note,
                    width=defaults.PRINT_WRAP_WIDTH,
                    break_long_words=False,
                    break_on_hyphens=False,
                    initial_indent="  - ",
                    subsequent_indent="    ",
                )
            )
```